pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -244,7 +244,8 @@ public static void main(String[] args) throws Exception {
         // Format BookKeeper ledger storage metadata
         if (arguments.existingBkMetadataServiceUri == null && arguments.bookieMetadataServiceUri == null) {
             ServerConfiguration bkConf = new ServerConfiguration();
-            bkConf.setMetadataServiceUri("metadata-store:" + metadataStoreUrlNoIdentifer);
```

Comment:
Nice catch!

I think it should be another BUG here? If we removed the identifier,  the `MetadataStoreExtended` will always use zookeeper?

Reply:
Yes,  So I remain  the identifier.